### PR TITLE
Feature/enable null urls

### PR DESCRIPTION
**Pull request by fivetran-catfritz**
**Are you a current Fivetran customer?** 

Fivetran created PR. 
**What change(s) does this PR introduce?** 

- For use in the [dbt_ad_reporting package](https://github.com/fivetran/dbt_ad_reporting), users can now allow records having nulls in url fields to be included in the `ad_reporting__url_report` model.
- Disabled the `not_null` test for `snapchat_ads__url_report` when null urls are allowed

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [x] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [x] Yes, Issue/Feature [#59](https://github.com/fivetran/dbt_ad_reporting/issues/59)
- [ ] No 

**How did you test the PR changes?** 

 

- [x] BuildKite  
- [x] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [x] BigQuery
- [x] Redshift
- [x] Snowflake
- [x] Postgres
- [x] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🍝 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

**Review by fivetran-sheringuyen — APPROVED** [review]
lgtm! 

feel free to merge once buildkite passes & docs have been regenerated

File: models/snapchat.yml [hunk 1]
```diff
@@ -151,6 +151,11 @@ models:
         description: The current used by the account in Snapchat.
       - name: base_url
         description: The base URL of the ad, extracted from the web_view_url.
+        tests:
+          - not_null:
+              config:
+                # Below is so we don't run this test if the user has allowed urls to have null entries
+                enabled: "{{ var('ad_reporting__url_report__using_null_filter', True) }}"
```
Comment (fivetran-sheringuyen) [inline models/snapchat.yml:158]: i believe this config needs to also appear in the unique combo test as well

**Review by fivetran-sheringuyen — APPROVED** [review]
hey @fivetran-catfritz, looks. Just need one more update inside the yml re testing. 

But once you add this feel free to merge once buildkite passes & docs have been regenerated

File: models/snapchat.yml (hunk 1 above)
Comment (fivetran-catfritz) [reply models/snapchat.yml:161]: updated.

**Pull request merged**